jenkins/PerfCI-Chaos/05-PerfCI-Chaos-Verify-Windows-VMs-Deployment/Jenkinsfile

```diff
@@ -0,0 +1,222 @@
+        PROVISION_IP = credentials('perfci_provision_ip')
+        SCALE_NODES = credentials('perfci_scale_nodes')
+        PIN_NODE_BENCHMARK_OPERATOR = credentials('perfci_pin_node_benchmark_operator')
+        PIN_NODE1 = credentials('perfci_pin_node1')
+        PIN_NODE2 = credentials('perfci_pin_node2')
+        ELASTICSEARCH = credentials('perfci_elasticsearch')
+        ELASTICSEARCH_PORT = credentials('perfci_elasticsearch_port')
+        IBM_REGION_NAME = credentials('perfci_ibm_region_name')
+        IBM_ENDPOINT_URL = credentials('perfci_ibm_endpoint_url')
+        IBM_ACCESS_KEY_ID = credentials('perfci_ibm_access_key_id')
+        IBM_SECRET_ACCESS_KEY = credentials('perfci_ibm_secret_access_key')
+        IBM_BUCKET = credentials('perfci_ibm_bucket')
+        IBM_KEY = credentials('perfci_ibm_key')
+        RUN_ARTIFACTS_URL = credentials('perfci_run_artifacts_url')
+        REDIS = credentials('perfci_redis')
+        WORKER_DISK_IDS = credentials('perfci_worker_disk_ids')
+        WINDOWS10_URL = credentials('perfci_windows10_url')
+        WINDOWS11_URL = credentials('perfci_windows11_url')
+        QUAY_BENCHMARK_RUNNER_REPOSITORY = credentials('perfci_quay_repository')
+        WINDOWS_SERVER_2019_URL = credentials('perfci_windows_server_2019_url')
+        WINDOWS_SERVER_2022_URL = credentials('perfci_windows_server_2022_url')
+        LSO_NODE = credentials('perfci_lso_node')
+        KUBEADMIN_PASSWORD = readFile('/home/jenkins/.kube/kubeadmin-password').trim()
+        KUBEADMIN_PASSWORD_PATH = '/home/jenkins/.kube/kubeadmin-password'
+        KUBECONFIG_PATH = '/root/.kube/config'
+        PRIVATE_KEY_PATH = '/home/jenkins/.ssh/provision_private_key'
+        CONFIG_PATH = '/home/jenkins/.ssh/config'
+        WORKSPACE='/home/jenkins'
+        CONTAINER_PRIVATE_KEY_PATH = '/root/.ssh/provision_private_key'
+        CONTAINER_CONFIG_PATH = '/root/.ssh/config'
+        CONTAINER_KUBECONFIG_PATH = '/home/jenkins/.kube/config'
+        WORKER_DISK_PREFIX = 'wwn-0x'
+        SAVE_ARTIFACTS_LOCAL = 'False'
+        ENABLE_PROMETHEUS_SNAPSHOT = 'True'
+        DELETE_ALL = 'False' // Not delete the running Windows11 VMs
+        VERIFICATION_ONLY = 'True'
+        RUN_TYPE = 'test_ci'
+        PROVISION_PORT = 22
+        NUM_ODF_DISK = 6
+        WINDOWS_SCALE = 50
+        THREADS_LIMIT= 20
+        PROVISION_TIMEOUT = 3600
+        TIMEOUT = 8000
+    }
+    stages {
+        stage('Cleanup') {
+            steps {
+                script {
+                    // Clean the Jenkins workspace
+                        echo "Cleaning Jenkins workspace"
+                        deleteDir()
+                    sh '''
+                        if [[ "$(sudo podman images -q ${QUAY_BENCHMARK_RUNNER_REPOSITORY} 2> /dev/null)" != "" ]]; then
+                            sudo podman rmi -f $(sudo podman images -q ${QUAY_BENCHMARK_RUNNER_REPOSITORY} 2> /dev/null)
+                        fi
```

Comment:
See above about running containers.

Reply:
The main purpose of this deletion is to remove all running containers of benchmark-runner to allow a fresh deployment.